scripts/mppc_to_multiprobe.py

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/python
+# -*- encoding: utf-8 -*-
+"""
+Created on 22 Aug 2019
+
+@author: Sabrina Rossberger, Delmic
+
+This script provides a command line interface for aligning of the MPPC detector to multiprobe.
+It returns the necessary corrections that need to be applied to the hardware (angle to rotate the mppc,
+offset in x/y to adjust the heights). For this purpose it is necessary to acquire an image in the below
+described manner and analyze the acquired image with this script.
+
+Preparation for imaging:
+    On the SEM Control PC:
+        Increase the scan size (normally 3.2um) to cover the active area of the full mppc detector, by
+        setting the params to achieve a 1000x magnification.  # TODO correct? on full screen?
+        Set beam shift to zero. TODO Eventually it should be the case when system correctly aligned.
+        Select single-beam mode; use beamlet E4. Aperture 8, manually select correct beam.
+        Set the scan mode to ‘External’ (spot mode).
+        All mppc cells should be active.
+    For scan control:
+        Set galvo mirror deflection to zero; X = Y = 0 (descan) -> mirror_amp = 0 in config.csv
+        and run scan_generation.py to apply change to AWGs.
+    Configure the camera to acquire a single field image.
+
+The scanning is performed with one beamlet/single probe E4 using all mppc cells and the deflectors (ebeam scan).
+
+Correction on hardware:
+    The offset correction needs to be applied to the galvo mirror settings in the config.csv file.
+    Then run scan_generation.py to apply the new settings.
+    The rotation correction needs to be manually adjusted via the rotational mounting on the mppc detector.
+    TODO rotation matches the coordinate system of mppc with coordinate system of multiprobe
+
+"""
+
+from __future__ import division
+
+import argparse
+import logging
+import math
+import numpy
+
+import cv2
+import glob
+import os
+
+from skimage import measure
+
+from odemis.util.img import RGB2Greyscale
+
+from odemis.dataio import tiff
+from libtiff import TIFF
```

Comment:
crtl+alt+o in pycharm orders the imports alphabetically and based on local and outside imports.